.travis.yml

```diff
@@ -7,6 +7,7 @@ env:
     global:
     - BYOND_MAJOR="512"
     - BYOND_MINOR="1403"
+    - RUST_G_VERSION="0.2.0"
```

Comment:
You can set this to master if you want

Reply:
another trick is to use a branch called stable that you update to point at the same commit as the release tag when you do your stable releases.

saves having to update the travis env yml every time you release.

Reply:
I think I'll leave this for now, as the bundled .DLL already needs to be updated. If things change I'll likely rewrite the shell script to just pull the latest tag.